environment.yml

```diff
@@ -5,12 +5,6 @@ name: rte_rrtmgp_test
 
 dependencies:
   - conda-forge::python=3.9
-  - conda-forge::urllib3
   - conda-forge::netcdf4
   - conda-forge::xarray
   - conda-forge::dask
-  - conda-forge::numpy
```

Comment:
@skosukhin I understand that removing numpy etc. will streamline the CI environment, but then we can't make the validation plots in `tests` which we do with the `ifort` image. Could we instead create an `environment-noplots.yml`? Also please leave `numpy` as a dependency - it follows from `dask` and `xarray` but I'd rather be explicit

Reply:
The `ifort` image uses `pip` to install the plotting packages, so it would not be affected by these changes. I also thought that `environment-dev.yml` was supposed to deliver the full development environment. It looks like I misinterpreted the comment in `environment.yml`. Nonetheless, I followed your suggestion and added `environment-noplots.yml`. And yes, my bad, `numpy` should be explicitly listed since it is used directly (i.e. not via calls to some other package) in the check scripts.